app/models/vaccination_record.rb

```diff
@@ -139,6 +140,17 @@ class VaccinationRecord < ApplicationRecord
        },
        validate: true
 
+  attribute :source, :integer
+  enum :source,
+       {
+         mavis: 0,
```

Comment:
```suggestion
         service: 0,
```

Nitpick, but we generally don't refer to the name of the service inside the code as it keeps it generic. I think this naming follows closer with our existing `recorded_in_service` scope.